qiskit/primitives/base/base_primitive.py

```diff
@@ -73,14 +73,16 @@ def _validate_circuits(
 
     @staticmethod
     def _validate_parameter_values(
-        parameter_values: Sequence[Sequence[float]] | Sequence[float] | float | None,
-        default: Sequence[Sequence[float]] | Sequence[float] | None = None,
+        parameter_values: Sequence[Sequence[float] | Mapping[ParameterExpression, float]]
+        | Sequence[float]
+        | Mapping[ParameterExpression, float]
+        | float
+        | None,
+        circuits: tuple[QuantumCircuit, ...],
     ) -> tuple[tuple[float, ...], ...]:
-        # Allow optional (if default)
+        # Allow optional
         if parameter_values is None:
-            if default is None:
-                raise ValueError("No default `parameter_values`, optional input disallowed.")
-            parameter_values = default
+            parameter_values = [()] * len(circuits)
```

Comment:
I kind of liked this being here, as it allowed the function to be reused in other contexts (e.g. by other `Estimator`s). Is there any reason to remove it?

Reply:
I see. What is the other Estimator? And this is private API, so not fixed. Other estimators should not use this in general.